pkg/executor/operate_ddl_jobs.go

```diff
@@ -85,3 +93,126 @@ type PauseDDLJobsExec struct {
 type ResumeDDLJobsExec struct {
 	*CommandDDLJobsExec
 }
+
+// AlterDDLJobExec indicates an Executor for alter config of a DDL Job.
+type AlterDDLJobExec struct {
+	exec.BaseExecutor
+	jobID     int64
+	AlterOpts []*core.AlterDDLJobOpt
+}
+
+// Open implements the Executor Open interface.
+func (e *AlterDDLJobExec) Open(ctx context.Context) error {
+	newSess, err := e.GetSysSession()
+	if err != nil {
+		return err
+	}
+	defer e.ReleaseSysSession(kv.WithInternalSourceType(context.Background(), kv.InternalTxnDDL), newSess)
+
+	return e.processAlterDDLJobConfig(ctx, newSess)
+}
+
+func getJobMetaFromTable(
+	ctx context.Context,
+	se *sess.Session,
+	jobID int64,
+) (*model.Job, error) {
+	sql := fmt.Sprintf("select job_meta from mysql.%s where job_id = %s",
+		ddl.JobTable, strconv.FormatInt(jobID, 10))
+	rows, err := se.Execute(ctx, sql, "get_job_by_id")
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	if len(rows) == 0 {
+		return nil, fmt.Errorf("ddl job %d not found", jobID)
```

Comment:
```suggestion
		return nil, fmt.Errorf("ddl job %d is not running", jobID)
```

Reply:
Yes, I see it. But I have an unrelated naive question: If we get a job_meta from mysql.ddl_job_table, Is the job state always running or may be some other state like pending、canceling ...

Reply:
Actually I want to ask if we should check the job state

Reply:
We don't need to check the state, because there is no harm in changing job_meta even if they are pending or canceling.